packages/KCard/KCard.vue

```diff
@@ -3,14 +3,22 @@
     :class="[borderVariant, {'hover': hasHover, 'kcard-shadow': hasShadow }]"
```

Comment:
I think the wrapper element of the card is a good use case for the `section` element https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section